tests/python/rest_api/test_projects.py

```diff
@@ -458,6 +458,23 @@ def test_admin_can_get_project_backup_and_create_project_by_backup(self, admin_u
             )
             assert response.status == HTTPStatus.ACCEPTED
 
+    def test_can_import_export_dataset_with_imagenet_format(self):
+        # https://github.com/opencv/cvat/issues/4850
+        username = "admin1"
+        format_name = "ImageNet 1.0"
+        project_id = 4
+
+        response = self._test_export_project(username, project_id, format_name)
+
+        tmp_file = io.BytesIO(response.data)
+        tmp_file.name = "dataset.zip"
+
+        import_data = {
+            "dataset_file": tmp_file,
+        }
+
+        self._test_import_project(username, project_id, format_name, import_data)
+
```

Comment:
Probably, this test should be removed when #4544 is merged.

Reply:
I think this test is needed. It refers to a specific issue. I agree that the updated test in the #4544 covers it. But the test as here is the most understandable and transparent (as it seems to me)